Viper/Viper/CountryListInteractor/CountryListInputProtocol.swift

```diff
@@ -0,0 +1,11 @@
+// CountryListInputProtocol.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Foundation
+
+/// Протокол инпут интерактора
+protocol CountryListInteractorInputProtocol: AnyObject {
+    var networkService: NetworkServiceProtocol { get set }
+    var presenter: CountryListInteractorOutputProtocol? { get set }
```

Comment:
```suggestion
    var networkService: NetworkServiceProtocol { get set }
    var presenter: CountryListInteractorOutputProtocol? { get set }
```
проверь точно ли нужен set?